api/envoy/config/filter/http/rbac/v2/rbac.proto

```diff
@@ -11,19 +11,30 @@ import "gogoproto/gogo.proto";
 // [#protodoc-title: RBAC]
 // Role-Based Access Control :ref:`configuration overview <config_http_filters_rbac>`.
 
+// RBAC filter config.
+// User sees the shadow rules as replacement to the enforced rules.
+// For example, policy A runs in prod in enforced mode, user wants
+// to test policy B; in the config, rules contains {A.mode=enforced},
```

Comment:
I think this text is out of data for an old implementation? It doesn't make sense. I think you can just remove and leave to the documentation below to cover.

Reply:
removed.